pkgs/development/python-modules/click-datetime/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchFromGitHub
+, click }:
+
+buildPythonPackage rec {
+  pname = "click-datetime";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "click-contrib";
+    repo = pname;
+    rev = "0.2.0";
```

Comment:
```suggestion
    rev = version;
```

Reply:
I will migrate to fetchPypi to remove the confusion.

Reply:
Except I moved to fetchFromGitHub since they don't ship the README in the package up on pypi.

```Executing setuptoolsBuildPhase
Traceback (most recent call last):
  File "nix_run_setup", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 5, in <module>
    with open('README.md') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
builder for '/nix/store/8bg983isbyrrkvbzncdvs7a3fgacs6dj-python3.8-click-datetime-0.2.drv' failed with exit code 1
```

So I think this is ok as is.